Sources/GRPCReflectionService/Server/ReflectionService.swift

```diff
@@ -21,9 +21,15 @@ import SwiftProtobuf
 
 @available(macOS 10.15, iOS 13, tvOS 13, watchOS 6, *)
 public final class ReflectionService: CallHandlerProvider, Sendable {
-  private let reflectionService: ReflectionServiceProvider
+  private let reflectionServiceProvider: ProviderType
+
   public var serviceName: Substring {
-    self.reflectionService.serviceName
+    switch self.reflectionServiceProvider {
+    case .v1Provider(let reflectionV1Provider):
```

Comment:
Same with this naming: we're switching over providers so each case doesn't need to have "provider" in the name, "v1" and "v1Alpha" are fine. Same goes for the associated value labels, `provider` is fine there too.

```swift
switch self.provider {
case .v1(let provider):
  // ...
case .v1Alpha(let provider):
  // ...
}
```